go/cmd/generator/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/goagent/config/go/generator
+
+go 1.15
```

Comment:
As much as I love being on the latest and greatest, we need to ensure that we work with older versions of go, as not all clients may be on 1.15 yet.

Reply:
This is actually tooling so we don't expect users to run this version of go. Instead whenever doing changes to the proto definition we have to run this.